polygon-daemon/templates/deployment.yaml

```diff
@@ -0,0 +1,66 @@
+{{- range $part := .Values.parts }}
```

Comment:
Instead of this pattern did you consider placing both containers in one pod?

Reply:
I've placed both containers in one pod. Thank you